CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Local Payment
```

Comment:
I know historically we've added something along the lines of "fixes Google Play store blah blah violation". May be helpful here for merchants/internal folks to see what version fixes this.

Reply:
Good call, let me add a bullet for that. Would it make sense to make these bullets a sub-point under the "fixes Google Play Store ..." bullet?

Reply:
Yeah I would say we should mirror whatever wording we are going to put in the dev docs in the changelog as well something like 
```
* Fixes Google Play Store Rejection
  * See [developer documentation](link) for required updates for user location data compliance
```

Reply:
What Sarah said makes sense to me!

Reply:
Sounds good. I've updated the wording in the PR. Will have a follow up PR once we get the approved copy.